examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -62,8 +62,11 @@
 ###############################################################################
 # Estimate second level model
 # ---------------------------
-# We define the input maps and the design matrix for the second level model
-# and fit it.
+# We wish to perform a one-sample test.
+# In order to do so, we need to create a design matrix that determines how
+# the analysis will be performed.
+# For a one-sample test, all we need to include in the design matrix is a
+# single column of ones, corresponding to the model intercept.
```

Comment:
Sounds great. I would simply add:
the one-sample test assumes that the distribution is symmetric about 0 (which is is weaker than assuming that it is Gaussian and centered about 0).

Reply:
Are you referring to both `SecondLevelModel` and `permuted_ols`, or just one of them?

Reply:
Sorry, `permuted_ols`

Reply:
WDYT of this?

```python
# .. important::
#   One key difference between
#   :obj:`~nilearn.glm.second_level.SecondLevelModel` and
#   :func:`~nilearn.glm.second_level.non_parametric_inference`/
#   :func:`~nilearn.mass_univariate.permuted_ols`
#   is that the one-sample test in non_parametric_inference/permuted_ols
#   assumes that the distribution is symmetric about 0,
#   which is is weaker than the SecondLevelModel's assumption that
#   the null distribution is Gaussian and centered about 0.
```

Reply:
Yes, thx !

Reply:
I've added it. Thank you.